OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -343,6 +351,24 @@ def enable():
 
     return exit_code
 
+
+def remove_workspace_configuration():
```

Comment:
Please add a comment about why this workaround is needed. something like this .."this is needed to distinguish between extension removal vs extension upgrade and to workaround waagent upgrade routine calling remove on old version before calling upgrade on new extension version issue.

Reply:
Done.